### ISSUE TITLE
When nimbits server is offline, nimbits client throws NPE

### ISSUE THREAD

**Issue by nikolchina** [bsautner/com.nimbits#209]
There should be a reasonable response instead of NPE as it was in previous versions of client.
We want to distinguish between different situations on server side to be able to react in appropriate way, so it is better to use different Exception objects. But it will be sufficient for now, if you just return previous behaviour, when exception messages are different.

**Comment by nikolchina** [comment]
java.lang.NullPointerException
        at com.nimbits.client.io.Nimbits.findPointByName (Nimbits.java:327)
        at com.nimbits.client.io.Nimbits.recordValueSync (Nimbits.java:450)

**Comment by nikolchina** [comment]
on client side we need to distinguish between "nimbits is unavailable" and "point not found", we use this piece of code
```
try {
      client.recordValueSync(dataPoint, nimbitsValue);
} catch (Throwable th) {
      LOGGER.error("Exception while sending event to nimbits", th);
      if (th.getMessage().contains("Not Found")) {
          // skip
      } else  {
          // save to DB
     }
}
```
Make sure, that this part of the message is not changed and we will still be able to distinguish.

**Comment by bsautner** [comment]
fixed in 4.1.5
